ambry-server/src/main/java/com.github.ambry.server/AmbryRequests.java

```diff
@@ -606,6 +607,33 @@ private void handleAdminRequest(Request request) throws InterruptedException, IO
             }
           }
           break;
+        case ReplicationControl:
+          error = ServerErrorCode.No_Error;
+          metrics.replicationControlRequestQueueTimeInMs.update(requestQueueTime);
```

Comment:
Yeah - just following convention where each request type in the server has its own time tracking (Get Blob, Get BlobInfo etc)